xml/System.Runtime.Intrinsics/Vector128.xml

```diff
@@ -18,7 +18,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides a collection of static convenience methods for creating, manipulating, combining, and converting generic 128-bit vectors.</summary>
```

Comment:
Same as https://github.com/dotnet/runtime/pull/76687

Combining and converting are a small subset of the total operations supported. Maybe generalizing this as the following would be better:
> Provides a collection of static methods for creating, manipulating, and otherwise operating on 128-bit vectors.